pkg/relay/ethereum/checkpoint.go

```diff
@@ -0,0 +1,62 @@
+package ethereum
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+)
+
+type checkpointType string
+
+const (
+	sendCheckpoint checkpointType = "send"
+	recvCheckpoint checkpointType = "recv"
+)
+
+func checkpointFileName(cpType checkpointType) string {
+	return string(cpType) + ".cp"
+}
+
+func (c *Chain) ensureDataDirectory() (string, error) {
+	path := filepath.Join(c.homePath, "ethereum")
```

Comment:
In cases where two different ethereum chain instances are used (e.g. relay between ethereum and bsc), don't their checkpoint locations conflict?

Reply:
Yes... I will fix the path. Thank you.

Reply:
Fixed: https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/6/commits/1c12faac4bdbe12f70ce1af3a57decb7976eb24d